packages/@react-spectrum/tooltip/stories/TooltipTrigger.stories.tsx

```diff
@@ -263,18 +263,6 @@ export const CrossoffsetExamples: TooltipTriggerStory = {
           <ActionButton>Tooltip Trigger -10</ActionButton>
           <Tooltip>Tooltip message.</Tooltip>
         </TooltipTrigger>
-        <TooltipTrigger delay={0} placement="left top" crossOffset={10}>
```

Comment:
Extraneous, mirrors the three tooltip triggers above them exactly except for the contents of the action button